### PR TITLE
Add docs for custom policy tagging

### PR DESCRIPTION
**Pull request by kilfoyle**
This adds docs for the new custom fields that be added to an agent policy. Thanks @kaanyalti for the super clear test instructions in your PR!

Target: 8.15
Closes: #1134

Screenshot 2024-07-15 at 2 09 31 PM

### CODE REVIEW

**Comment by kilfoyle** [comment]
Oops! I forgot that Kaan is on leave. :-)

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 1]
```diff
@@ -263,9 +267,48 @@ Alternatively, click **Delete policy** to delete the policy.
 Existing data is not deleted.
 Any agents assigned to a policy must be unenrolled or assigned to a different policy before a policy can be deleted.
 
+[discrete]
+[[add-custom-fields]]
+== Add custom fields
+
+Use this setting to add a custom field and value set to all data collected from the {agents} enrolled in an {agent} policy. This is useful when you want to identify or visualize all of the data from a group of agents, and possibly manipulate it downstream.
```
Comment (karenzone) [inline docs/en/ingest-management/agent-policies.asciidoc:274]:
```suggestion
Use this setting to add a custom field and value set to all data collected from the {agents} enrolled in an {agent} policy. 
Custom fields are useful when you want to identify or visualize all of the data from a group of agents, and possibly manipulate the data downstream.
```

Comment (karenzone) [reply docs/en/ingest-management/agent-policies.asciidoc:274]: Applies Asciidoctor's one sentence per line recommendation. 
Replaces  some pronouns with more descriptive words for better SEO and easier parsing.

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 2]
```diff
@@ -263,9 +267,48 @@ Alternatively, click **Delete policy** to delete the policy.
...
+. Click the **Settings** tab and scroll to **Custom fields**.
+
+. Click **Add field**.
+
+. Specify a field name and value.
++
+[role="screenshot"]
+image::images/agent-policy-custom-field.png[Sceen capture showing the UI to add a custom field and value]
+
+. Click **Add another field** for additional fields, or click **Save changes** when you're done.
```
Comment (karenzone) [inline docs/en/ingest-management/agent-policies.asciidoc:290]:
```suggestion
. Click **Add another field** for additional fields. Click **Save changes** when you're done.
```

Comment (karenzone) [reply docs/en/ingest-management/agent-policies.asciidoc:290]: Remove "or" to allow for either/or case.

File: docs/en/ingest-management/agent-policies.asciidoc [hunk 3]
```diff
@@ -263,9 +267,48 @@ Alternatively, click **Delete policy** to delete the policy.
...
+To edit a custom field:
+
+. In {fleet}, click **Agent policies**.
+Select the name of the policy you want to edit.
+
+. Click the **Settings** tab and scroll to **Custom fields**. Any custom fields that have been configured are shown.
+
+. Click the edit icon to update a field or click the delete icon to remove it.
+
+Note that for a small set of inputs, adding custom tags is not supported:
```
Comment (karenzone) [inline docs/en/ingest-management/agent-policies.asciidoc:301]:
```suggestion
Note that adding custom tags is not supported for a small set of inputs:
```

Comment (karenzone) [reply docs/en/ingest-management/agent-policies.asciidoc:301]: To keep description closer to the list.

**Review by karenzone — DISMISSED** [review]
Nice!  I left some minor suggestions inline for your consideration.  Either way, LGTM.

**Comment by kilfoyle** [comment]
Thanks @karenzone!

**Review by karenzone — APPROVED** [review]
LGTM! 🚀

**Pull request merged**